distribution/lib/Standard/Microsoft/0.0.0-dev/docs/api/SQLServer_Connection.md

```diff
@@ -24,6 +24,5 @@
     - tables self name_like:Standard.Base.Data.Text.Text= database:Standard.Base.Data.Text.Text= schema:Standard.Base.Data.Text.Text= types:Standard.Base.Any.Any= all_fields:Standard.Base.Any.Any= -> Standard.Base.Any.Any
     - to_js_object self -> Standard.Base.Any.Any
     - truncate_table self table_name:Standard.Base.Any.Any -> Standard.Base.Any.Any
-- make_table_name_selector connection:Standard.Base.Any.Any -> Standard.Base.Any.Any
```

Comment:
- An API change in `Standard.Microsoft` when introducing a completely unrelated DB support!
- a sign of an unwanted change error
- or sign of previously low attention to API design
- probably the second, `make_table_name_selector` should have never been in the API, right?

Reply:
Moving functionality that was added to the Microsoft library to the base library as generally reusable - reading from tables with schemas.